packages/account/src/Sections/Security/ConnectedApps/data-list-template-entry.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { Text } from '@deriv/components';
+import { Localize } from '@deriv/translations';
+
+type TDataListTemplateEntry = {
+    title: string;
+    content: string;
+};
+
+const DataListTemplateEntry = ({ title, content }: TDataListTemplateEntry) => {
+    const text_size = 'xxs';
+
+    return (
+        <React.Fragment>
+            <Text size={text_size} weight='bold'>
+                <Localize i18n_default_text={title} />
```

Comment:
does the translation work with this approach?
should we better pass the title and content with Localize?

Reply:
please refactor it, our translation will not work here